lib/tests/modules.sh

```diff
@@ -365,6 +365,9 @@ checkConfigError \
   config.set \
   ./declare-set.nix ./declare-enable-nested.nix
 
+# Check that that merging of option collisions doesn't depend on type being set
+checkConfigError 'The option .group..*would be a parent of the following options, but its type .<no description>. does not support nested options.\n - option.s. with prefix .group.enable..*' config.group.enable ./merge-typeless-option.nix
```

Comment:
This [appears not to match](https://gist.github.com/GrahamcOfBorg/3d3844076360b06bbe8c81a05b578a2a), against

``
error: The option `group' in module `/build/lib/tests/modules/merge-typeless-option.nix' would be a parent of the following options, but its type `<no description>' does not support nested options.
        - option(s) with prefix `group.enable' in module `/build/lib/tests/modules/merge-typeless-option.nix'
``

Maybe it's because of the whitespace before `-`?

Reply:
Nix > 2.3 inserts extra whitespace, should be accounted for now.